vendingMachine/vendingMachineV3/src/interfaces/UserFunction.java

```diff
@@ -1,6 +1,6 @@
 package interfaces;
 
-public interface UserVendingMachine {
+public interface UserFunction {
   void insertCoinForPayment();
   void selectProduct();
   void display();
```

Comment:
Display is not User function I think. I Feel Like it should not be here @vineethreddyp

Reply:
Yep. makes sense. Will move this to Display Panel